io/driver/driver_bind_test.py

```diff
@@ -41,37 +41,47 @@ def setUp(self):
         Setup the device.
         """
         self.return_code = 0
-        self.slot = self.params.get('pci_device', default='0001:01:00.0')
-        self.driver = pci.get_driver(self.slot)
-        if not self.driver:
-            self.cancel("%s does not exist" % self.slot)
+        self.pci_dic = {}
+        self.pci_devices = self.params.get('pci_device', default=None)
+        self.count = self.params.get('count', default=1)
+        if not self.pci_devices:
+            self.cancel("No pci_adresses Given")
+        for pci_addr in self.pci_devices.split(","):
+            self.pci_dic[pci_addr] = pci.get_driver(pci_addr)
```

Comment:
@Naresh-ibm This dic is actually not needed. You can get the driver on the go in the for loop in `test()`, provided you make the order like as `iteration of count per device` instead of `iteration of device per count`.

Which brings the next logical suggestion: can the test be like:
```
for pci_addr in self.pci_devices.split(","):
    driver = pci.get_driver(pci_addr)
    for _ in range(self.count):
       ...
```

Reply:
sounds good. will change.
any other review comments? I will change in one shot.

Thanks

Reply:
That is it.